compiler/z/codegen/S390Instruction.cpp

```diff
@@ -2010,7 +2010,12 @@ TR::S390RILInstruction::adjustCallOffsetWithTrampoline(int32_t offset, uint8_t *
 
    // Check to make sure that we can reach our target!  Otherwise, we need to look up appropriate
    // trampoline and branch through the trampoline.
+   
+#if defined(TR_TARGET_64BIT) && defined(STRESS_TRAMPOLINES)
+   if (1)
```

Comment:
I like the idea of stressTrampolines compiler option, should be a simple change from if(X) to if(getOption(stressTrampoline) OR X). I will talk with Joran to confirm.